src/main/java/io/zksync/wallet/DefaultZkSyncWallet.java

```diff
@@ -63,43 +71,36 @@ public String setSigningKey(TransactionFee fee, Integer nonce, boolean onchainAu
     }
 
     @Override
-    public String syncTransfer(String to, String tokenIdentifier, BigInteger amount, BigInteger fee, Integer nonce) {
+    public String syncTransfer(String to, BigInteger amount, TransactionFee fee, Integer nonce) {
```

Comment:
In `zksync.js` and `zksync.rs` methods such as `syncTransfer` do not return the hash of the transactions. The return a special object that has functions for conveniently waiting until the transactions are either committed or veriified.

[Example in zksync.js](https://github.com/matter-labs/zksync/blob/5b936b1855a08033cca7f75d6f87fde106c6e8fd/sdk/zksync.js/src/wallet.ts#L792). It has the `awaitReceipt` function.
[Example in zksync.rs](https://github.com/matter-labs/zksync/blob/5b936b1855a08033cca7f75d6f87fde106c6e8fd/sdk/zksync-rs/src/operations/mod.rs#L32). It has the `wait_for_commit` function. 

But I am not sure if that is inside of the scope of this PR. @popzxc could you please clarify?

Reply:
I think for the very first release it is OK to return the transaction hash. This is a quality-of-life topic, of course, but the point is to provide at least *some* interface to interact with zkSync. Improvements can be added after.

Reply:
I already developing transaction processor based on web3j, but for first release I've decided left this as is